AppierExtension.xcframework/ios-arm64_x86_64-simulator/AppierExtension.framework/Modules/AppierExtension.swiftmodule/arm64-apple-ios-simulator.private.swiftinterface

```diff
@@ -1,6 +1,7 @@
 // swift-interface-format-version: 1.0
-// swift-compiler-version: Apple Swift version 5.10 (swiftlang-5.10.0.13 clang-1500.3.9.4)
+// swift-compiler-version: Apple Swift version 6.0 effective-5.10 (swiftlang-6.0.0.7.6 clang-1600.0.24.1)
```

Comment:
Will this change impact xcode15 or below?

Reply:
No, this change should not impact Xcode 15 or below. The framework is compiled using the Swift 5.10 standard, which is fully supported in Xcode 15 and earlier. Although the compiler in Xcode 16 supports Swift 6.0, the framework remains backward-compatible because it is compiled in a mode (effective-5.10) that ensures it behaves according to Swift 5.10 standards. As a result, the change will not affect older versions of Xcode that support Swift 5.10.